libs/shared/flagd-core/src/lib/targeting/fractional.ts

```diff
@@ -29,12 +29,12 @@ export function fractionalFactory(logger: Logger) {
       bucketBy = args[0];
       buckets = args.slice(1, args.length);
     } else {
-      bucketBy = `${flagdProperties[flagKeyPropertyKey]}${context[targetingPropertyKey]}`;
-      if (!bucketBy) {
+      const targetingKey = context[targetingPropertyKey];
```

Comment:
This was the bug... the flag key is set automatically by flagd.

Reply:
Nice!